react/features/jaas/functions.ts

```diff
@@ -70,7 +69,7 @@ export async function sendGetDetailsRequest({ appId, baseUrl }: {
         }
 
         throw new Error('Request not successful');
-    } catch (err) {
+    } catch (err: any) {
```

Comment:
Isn't there a type for fetch error?

Reply:
TS requires that errors in catch clauses must be either `any` or `unknown`